drf_extra_fields/fields.py

```diff
@@ -80,7 +80,7 @@ def get_file_extension(self, filename, decoded_file):
         raise NotImplementedError
 
     def get_file_name(self, decoded_file):
-        return str(uuid.uuid4())[:12]  # 12 characters are more than enough.
+        return str(uuid.uuid4())[:13]  # 13 characters are more than enough. 12 Characters can trigger ablockers.
```

Comment:
Since you mentioned why this can't be 12 characters, can you please also add stackoverflow issue's link to here?. So that we can have a clear explanation of the problem.

Reply:
I updated my original post with the link.

Reply:
I meant to add here. Such as: `... 12 Characters can trigger ablockers. Please see https://stackoverflow.com/questions/57227131 for more information.`

In the feature if it's needed to change lenght of the file name, developer should now if the file name ends with `-ad[0-9]`, it can be blocked by adblockers.

Reply:
OK gotcha.

Reply:
So should I create another PR?

Reply:
No, you can push a new commit to your branch, maratumba:master.

Reply:
Done.

Reply:
Coding style test is failed. You can check the logs from [here](https://travis-ci.org/Hipo/drf-extra-fields/jobs/564341676#L478).

While fixing those can you please move comments to over `return` statement?

Reply:
Should be fixed.